src/streamlit_analytics2/firestore.py

```diff
@@ -36,42 +38,42 @@ def load(
         creds = service_account.Credentials.from_service_account_info(key_dict)
         db = firestore.Client(credentials=creds, project=firestore_project_name)
         col = db.collection(collection_name)
-        firestore_counts = col.document("counts").get().to_dict()
+        firestore_data = col.document("data").get().to_dict()
         if session_id is not None:
-            firestore_session_counts = col.document(session_id).get().to_dict()
+            firestore_session_data = col.document(session_id).get().to_dict()
     else:
         db = firestore.Client.from_service_account_json(service_account_json)
         col = db.collection(collection_name)
-        firestore_counts = col.document("counts").get().to_dict()
+        firestore_data = col.document("data").get().to_dict()
```

Comment:
@emigre459 also here

Reply:
I don't think it will break my downstream, just add a new document in Firestore that I'll have to merge with my old `counts` document. But I don't *think* I call `counts` directly anywhere except in manual analysis.

Can you just provide a kwarg with a default value of `"data"` for naming the aggregate counts document? That way folks can easily config their way to something non-breaking.

Reply:
Hmmm thank you for the inputs
Ill have to set it up and do some testing to understand the creation flow better. Havent touched the firestore stuff in a while